ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -216,23 +217,143 @@ public int getMaxTrailingPeers() {
     private int downloaderChainSegmentTimeoutsPermitted = 5;
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-downloader-chain-segment-size",
+        hidden = true,
+        defaultValue = "200",
+        paramLabel = "<INTEGER>",
+        description = "Distance between checkpoint headers (default: ${DEFAULT-VALUE})")
     private int downloaderChainSegmentSize = 200;
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-downloader-parallelism",
+        hidden = true,
+        defaultValue = "4",
+        paramLabel = "<INTEGER>",
+        description =
+            "Number of threads to provide to chain downloader (default: ${DEFAULT-VALUE})")
     private int downloaderParallelism = 4;
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-transactions-parallelism",
+        hidden = true,
+        defaultValue = "2",
+        paramLabel = "<INTEGER>",
+        description =
+            "Number of threads to commit to transaction processing (default: ${DEFAULT-VALUE})")
     private int transactionsParallelism = 2;
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-computation-parallelism",
+        hidden = true,
+        paramLabel = "<INTEGER>",
+        description =
+            "Number of threads to make available for bulk hash computations durring downloads (default: # of processors)")
     private int computationParallelism = Runtime.getRuntime().availableProcessors();
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-fast-sync-pivot-distance",
+        hidden = true,
+        defaultValue = "50",
+        paramLabel = "<INTEGER>",
+        description =
+            "Distance from initial chain head to fast sync target (default: ${DEFAULT-VALUE})")
     private int fastSyncPivotDistance = DEFAULT_PIVOT_DISTANCE_FROM_HEAD;
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-fast-sync-full-validation-rate",
+        hidden = true,
+        defaultValue = "0.1",
+        paramLabel = "<FLOAT>",
+        description =
+            "Fraction of headers fast sync will fully validate (default: ${DEFAULT-VALUE})")
     private float fastSyncFullValidationRate = DEFAULT_FULL_VALIDATION_RATE;
+
     private int fastSyncMinimumPeerCount = DEFAULT_FAST_SYNC_MINIMUM_PEERS;
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-world-state-hash-count-per-request",
+        hidden = true,
+        defaultValue = "348",
+        paramLabel = "<INTEGER>",
+        description =
+            "Fast sync world state hashes queried per request (default: ${DEFAULT-VALUE})")
     private int worldStateHashCountPerRequest = DEFAULT_WORLD_STATE_HASH_COUNT_PER_REQUEST;
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-world-state-request-parallelism",
+        hidden = true,
+        defaultValue = "10",
+        paramLabel = "<INTEGER>",
+        description =
+            "Number of threads committed to fast sync world state downloading (default: ${DEFAULT-VALUE})")
```

Comment:
```suggestion
            "Number of concurrent requests to use when downloading fast sync world state (default: ${DEFAULT-VALUE})")
```

There's only ever one thread used, just multiple concurrent requests.